lib/puppet/modulebuilder/builder.rb

```diff
@@ -5,15 +5,29 @@
 module Puppet::Modulebuilder
   # Class to build Puppet Modules from source
   class Builder
-    DEFAULT_IGNORED = [
+    # Due to the way how PathSpec generates the regular expression,
+    # `/*` doesn't match directories starting with a dot,
+    # so we need `/.*` as well.
+    IGNORED = [
+      '/*',
```

Comment:
I think this needs to be recursive, because otherwise it included `junit/*` for me.
```suggestion
      '/**',
```